presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcDeletedRows.java

```diff
@@ -193,7 +206,12 @@ private void loadValidPositions()
                         Page page = pageSource.getNextPage();
                         if (page != null) {
                             for (int i = 0; i < page.getPositionCount(); i++) {
-                                deletedRowsBuilder.add(new RowId(page, i));
+                                long originalTransaction = -1;
+                                int bucket = -1;
+                                long row;
+                                originalTransaction = BIGINT.getLong(page.getBlock(ORIGINAL_TRANSACTION_INDEX), i);
+                                row = BIGINT.getLong(page.getBlock(BUCKET_ID_INDEX), i);
```

Comment:
```suggestion
                                row = BIGINT.getLong(page.getBlock(ROW_ID_INDEX), i);
```

Reply:
As now, we are reading only originalTxnId, rowId. So, we should read index 0, 1. As, 
BUCKET_ID_INDEX = 1, so I used it. I will change the variable value if this approach looks fine.